src/Symfony/Component/DependencyInjection/Tests/ContainerBuilderTest.php

```diff
@@ -307,6 +307,35 @@ public function testDeprecatedAlias()
         $builder->get('foobar');
     }
 
+    /**
+     * @group legacy
+     * @expectedDeprecation Since symfony/dependency-injection 5.1: The "foobar" service alias is deprecated. You should stop using it, as it will be removed in the future.
```

Comment:
we should change the message to make it clear that this test case should NOT be removed in 6.0
(same in similar cases below)